packages/react-components/src/util/toFormatedBalance.ts

```diff
@@ -0,0 +1,29 @@
+import { formatBalance } from '@polkadot/util';
+
+const toFormatedBalance = (value: string): string => {
```

Comment:
`POWER` could be a parameter with default value

Reply:
Definitely!